src/main/webapp/js/adminHome.es6

```diff
@@ -16,18 +16,41 @@
  * @param {String} status
  * @returns {String} a HTML row of action result table
  */
+ /**
+* Substring used to check if join link was created
+*/
+const successStatus = 'successfully created with join link:';
```

Comment:
This does not look like it's used outside the function `createRowForResultTable`. Move it inside the function?